release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -614,6 +614,16 @@ You can run cluster workloads on {oci-first} infrastructure that supports dedica
 {oci} provides services that can meet your needs for regulatory compliance, performance, and cost-effectiveness. You can access {oci} Resource Manager configurations to provision and configure {oci} resources.
 
 For more information, see xref:../installing/installing_oci/installing-oci-assisted-installer.adoc[Using the Assisted Installer to install a cluster on {oci}].
+[id="ocp-4-15-installation-oci-agent-based-installer-vm"]
+==== Using the Agent-based Installer to install a cluster on {oci} (Technology Preview)
+
+You can use the Agent-based Installer to install a cluster on {oci-first}, so that you can run cluster workloads on infrastructure that supports dedicated, hybrid, public, and multiple cloud environments.
+
+The Agent-based installer provides the ease of use of the Assisted Installation service, but with the capability to install a cluster in either a connected or disconnected environment.
+
+{oci} provides services that can meet your regulatory compliance, performance, and cost-effectiveness needs. {oci} supports 64-bit `x86` instances and 64-bit ARM instances.
+
+For more information, see xref:../installing/installing_oci/installing-oci-agent-based-installer.adoc[Using the Agent-based Installer to install a cluster on {oci}].
```

Comment:
same comment as the other PR that I reviewed (#71731 ), the title is slightly different than the link text. I assume that's intentional but marking just in case.